index.html

```diff
@@ -49,13 +65,22 @@ <h3 id="message"></h3>
         }
       }
 
+      function enableClear() {
+        let guessField = document.getElementById('user-guess');
+        let clearButton = document.getElementById('clear');
+        if (parseInt(guessField.value) == '') {
```

Comment:
Something about this condition isn't working because of the `parseInt` - since you don't care about the value, just an existence of a value, I'd just remove the `parseInt` that is wrapped around the `guessField.value`, and I think it should work!

Reply:
thank you! I need to review the JS debugging strategies I learned last week from my mentor so I can see the return values and not be debugging blind as I tried to do today.